resharper/test/data/CSharp/CodeCompletion/List/EditorWindow01.cs

```diff
@@ -0,0 +1,7 @@
+using UnityEngine;
+using UnityEditor;  // More types in scope
+
+public class A : EditorWindow
```

Comment:
Just add a quick comment here that this class inherits `Awake` from both `EditorWindow` and `ScriptableObject`.